collect.go

```diff
@@ -0,0 +1,59 @@
+package goshopify
+
+import (
+	"fmt"
+	"time"
+)
+
+const collectsBasePath = "admin/collects"
+const collectsResourceName = "collects"
+
+// CollectService is an interface for interfacing with the collect endpoints
+// of the Shopify API.
+// See: https://help.shopify.com/api/reference/product/collect
```

Comment:
that url is invalid, should be https://help.shopify.com/en/api/reference/products/collect

Reply:
good catch!